src/asm/symbol.c

```diff
@@ -720,5 +720,12 @@ void sym_Init(time_t now)
 	labelScope = NULL;
 	anonLabelID = 0;
 
-	math_DefinePI();
+	/* _PI is deprecated */
+	struct Symbol *_PISymbol = createsymbol("_PI");
```

Comment:
Shouldn't it be defined as a builtin symbol?

Reply:
Builtins can't be purged. When `_PI` is removed people will be able to define that name themselves, so this way they can purge the deprecated one and use the name.

Reply:
Doesn't `PURGE` error out when the argument symbols aren't defined?

Reply:
Yes; if you wanted the same source to work with the deprecated-pi *and* deleted-pi rgbasms, you'd have to do `if def(_PI) purge _PI endc` before defining `_PI`.

Reply:
I'm not a fan of encouraging people to use an underscore-prefixed symbol name. This would also work:
```asm
if def(_PI)
PI equ _PI
else
PI equ 3.14159
endc
```

Reply:
Alright, it's now still a built-in symbol.